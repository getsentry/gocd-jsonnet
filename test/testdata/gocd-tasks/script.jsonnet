local gocdtasks = import '../../../src/gocd-tasks.libsonnet';

{
  tasks: [
    gocdtasks.script("echo 'hello'"),
    gocdtasks.script(importstr '../sample-bash.sh'),
  ],
}
