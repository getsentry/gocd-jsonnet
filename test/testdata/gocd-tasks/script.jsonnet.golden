{
   "tasks": [
      {
         "script": "echo 'hello'"
      },
      {
         "script": "##!/bin/bash\n\n## A simple echo command\necho \"world\";\n"
      }
   ]
}
